src/eva/data/mon_data_space.py

```diff
@@ -300,52 +302,66 @@ def read_ieee(self, file_name, coords, dims, ndims_used, dims_arr, nvars, vars,
 
         filename = os.path.join(file_path, file_name) if file_path else file_name
 
+        load_data = True
         if not os.path.isfile(filename):
```

Comment:
nit pick, but I would prefer the `if` be flipped and not do an `if not` and then `else` but rather `if` then `else`

Reply:
Fair point, change made.